### DISCUSSION TITLE
New files sidebar should be closed by default because it's extremely visually disorienting and cluttered

### DISCUSSION THREAD

**Discussion by ell1e** [community/community#68234] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

The new files sidebar to the left showing a tree of the project when looking at a single file should be closed by default because it's extremely visually disorienting and cluttered. Same for the symbols side bar to the right. It's not that these new features aren't useful, but jumping them on every new visitor like that makes it hard to see what you're even looking at, which especially for just linking a small code section for reference for discussion or linking some specific markdown document quote is an extremely bad idea.

Basically, all these side bars seem to assume I use GitHub as an IDE. But most links will be for reference for discussions, not to develop using GitHub's Web IDE where I would want to browse around a lot, see the symbols and references prominently, etc. So these things should all be collapsed and hidden until people actually bring them up.

One of GitHub's strengths especially over Gitlab used to be its focused UI. You really tossed that out of the window by showing these panels by default which I think is a big regression. First impressions especially for end users (people not maintaining the project but rather coming in from the outside) are pretty vital.

**Comment by ell1e** [comment]
Basically I'm seeing some variant of this as a first time visitor or guest, not logged in:

![Screenshot from 2023-09-25 17-22-45](https://github.com/community/community/assets/64124388/7379fe57-72e2-4d58-8cb3-bbba5dc33c1b)


And I suggest that I should always just be seeing this:

![Screenshot from 2023-09-25 17-22-36](https://github.com/community/community/assets/64124388/9298f9fa-9884-4221-be97-6c6d35e61c0c)

**Comment by gbons002** [comment]
I agree with https://github.com/ell1e
